src/mame/video/k051960.h

```diff
@@ -88,7 +88,7 @@ class k051960_device : public device_t, public device_gfx_interface, public devi
 
 	uint8_t    m_spriterombank[3];
 	int      m_romoffset;
-	int      m_spriteflip, m_readroms;
+	int      m_spriteflip, m_readroms, m_shadow_config;
```

Comment:
You really shouldn’t be registering things for save states if they aren’t explicitly sized types.  I realise this device is already doing it with existing `int` members, but that’s not an excuse to add more of them.